### DISCUSSION TITLE
essay/2024/02/20/50000km

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#176] [Comments]
# essay/2024/02/20/50000km

어릴 적 저희 집에는 차가 없었습니다. 승용차를 타는 일이 거의 없었는데.. 가끔 누군가의 차를 얻어타면 그렇게 좋았습니다. 온 세상이 신기한 듯 창밖을 내다봤습니다.

https://jeho.page/essay/2024/02/20/50000km.html

**Comment by lovesich** [comment]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 얼간이가 5만이나 타다니. 기특하다. 사진 첨부하고 싶다.. 얼간이 처음 드라이브한다고 폼잡았던거 ㅋㅋㅋ
